src/plugins/analysis/information_leaks/code/information_leaks.py

```diff
@@ -58,6 +59,14 @@
 }
 
 
+def _filter_files_from_summary(path: str) -> str:
+    """if files are in the path list only return the parent directory for the summary"""
```

Comment:
[PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/) clearly states to "always use triple double quotes" for docstrings. Do we want to obey that PEP or do we keep our "single quotes only" convention?

Reply:
> [PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/) clearly states to "always use triple double quotes" for docstrings.

Interesting I didn't know.

Well, I personally don't like single quotes anyways so if I was to dictate the rules it would be double quotes only.
So lets leave it as double quotes.

With that being said we should probably `s/'''/"""/g` on the rest of the codebase tho.
I opened #894 to track this.

Reply:
> With that being said we should probably s/'''/"""/g on the rest of the codebase tho.
> I opened https://github.com/fkie-cad/FACT_core/issues/894 to track this.

While I personally would also like to do that, we should talk this over with @dorpvom